libs/datamodel/core/src/transform/ast_to_dml/db/walkers.rs

```diff
@@ -21,14 +21,6 @@ impl<'ast> ParserDatabase<'ast> {
             model_attributes: &self.types.model_attributes[&model_id],
         }
     }
-
-    pub(crate) fn walk_models(&self) -> impl Iterator<Item = ModelWalker<'ast, '_>> + '_ {
```

Comment:
Can you `#[allow(dead_code)]` on that one? We'll almost certainly need it back soon.